e2eTests/README.md

```diff
@@ -41,80 +26,60 @@ of how this setup works.
+for port in 5432 5672 5433 80 5673 15672 25672; do lsof -ti:$port | xargs kill -9; done
 ```
-</details>
-
-<details>
-  <summary>I made changes in `services/` but when I run latest are not there in the build</summary>
 
-Try removing the stale images.
+##### Remove stale images
 
 ```bash
 docker rmi tsd-proxy:latest tsd-api-mock:latest mq-interceptor:latest --force
 ```
-</details>
 
-<details>
-  <summary>I have `docker-compose` but I can't find `docker compose` command?</summary>
-
-In some cases, particularly on older Ubuntu distributions, you might find that you have `docker-compose` (V2) installed but not the `docker compose` subcommand. To resolve this, first ensure you have Compose V2 and then, you can create a symbolic link in the `cli-plugins` directory by executing the following commands:
-
-```bash
-mkdir -p ~/.docker/cli-plugins
-ln -sfn /usr/local/bin/docker-compose ~/.docker/cli-plugins/docker-compose
-```
+##### Docker Compose issues
 
-For migration guidance to Docker Compose, visit [migrate to docker compose](https://docs.docker.com/compose/migrate/). Also, for further discussions and troubleshooting, refer to the GitHub issue at https://github.com/docker/compose/issues/8630.
-</details>
+First, verify that `docker compose` is correctly installed. Ensure you have Compose V2.
 
-<details>
-  <summary>Docker fails to build mq-interceptor (dial tcp: lookup proxy.golang.org: i/o timeout) </summary>
+For migration guidance to Docker Compose, visit [migrate to docker compose](https://docs.docker.com/compose/migrate/).
 
-Do a docker daemon restart.
+In some cases, particularly on older Ubuntu distributions, you might find that you have `docker-compose` (V2) installed but not the `docker compose` subcommand. To resolve this, you can create a symbolic link in the `cli-plugins` directory by executing the following commands:
 
 ```bash
-sudo systemctl restart docker
+mkdir -p ~/.docker/cli-plugins
+ln -sfn /usr/local/bin/docker-compose ~/.docker/cli-plugins/docker-compose
 ```
-</details>
-
-<details>
-  <summary>I made a mess. I want to start from scratch.</summary>
 
-First, clean the e2eTests directory and all the submodules.
+For further discussions and troubleshooting, refer to the GitHub issue at https://github.com/docker/compose/issues/8630.
 
-```bash
-sudo rm -rf e2eTests/tmp e2eTests/bin docker-compose.yml &&
-./gradlew clean
-```
+##### Maybe try cleaning everything?
 
-And then let's prune the docker system and restarts the docker daemon.
+This prunes the docker system and restarts the docker daemon.
 
 ```bash
 docker stop $(docker ps -aq) && \
 docker rm $(docker ps -aq) && \
 docker rmi tsd-proxy:latest tsd-api-mock:latest mq-interceptor:latest --force && \
-lsof -ti:5432 -ti:5672 -ti:5433 -ti:80 -ti:5673 -ti:15672 -ti:25672 | xargs kill -9 \
+for port in 5432 5672 5433 80 5673 15672 25672; do lsof -ti:$port | xargs kill -9; done && \
 echo 'y' | docker system prune &&
 sudo systemctl restart docker
```

Comment:
We can consider using -15, which sends a SIGTERM signal. So, it allows processes to gracefully shut down. Or else what we can do it use entirely different ports in our local setup.

5434 (instead of 5432)
5674 (instead of 5672)
5435 (instead of 5433)
81 (instead of 80)
5675 (instead of 5673)
15673 (instead of 15672)
25673 (instead of 25672)

Reply:
Apologies if I wasn't clear with my comment. My issue is not with the specific ports being used or which kill signal is sent. Since this covers two issues I'll go over them in 2 points:

1. Stopping and removing all running containers -> This also forces containers that might be running completely unrelated to the the FEGA-Norway project. Imagine if someone was testing something on a container and let it run in the background for any reason. Now this person run this command without giving it too much thought and just lost a bunch of work. Same with a system prune. Any custom image created for any specific reason just got delete off their system.
2. Killing any process running on some ports. Regardless of what port it is, we have just causes a process that might not have been started by us to shut down. Imagine a scenario in which I have a whole lot of things running on my computer and some running on random ports, be it NGINX, some container I'm testing, some server I setup for experimentation etc. I do a fresh git clone on FEGA-Norway, try to run it, see that it doesn't work, run this code snippet; Now I have effectively stopped a process I was using and might not even know what on originally running on it. Again, this just leads to frustrations.

What I was suggesting is that we simply don't recommend this at all. At most we should say "should you have a port clash, either change the port on the docker compose file or change the port on whatever other process was using it"
And with the containers "Try removing the FEGA-Norway containers and images and re-running the `gradle` script/ `docker compose`" etc.

Hope that added some more clarity.

Reply:
I have updated the local setup guide under wiki pages.
https://github.com/ELIXIR-NO/FEGA-Norway/wiki/Local-Setup